packages/react-core/src/components/Spinner/Spinner.tsx

```diff
@@ -13,13 +13,15 @@ export interface SpinnerProps extends Omit<React.HTMLProps<HTMLSpanElement>, 'si
   /** Additional classes added to the Spinner. */
   className?: string;
   /** Size variant of progress. */
-  size?: 'sm' | 'md' | 'lg' | 'xl';
+  size?: 'sm' | 'md' | 'lg' | 'xl' | 'inline';
```

Comment:
is this necessary in addition to the inline property?

Reply:
I would agree with @gitdallas here.  It also seems odd to have `inline` as a size.

Reply:
D'oh! I tried this initially and forgot to take this out because it wasn't working. That said, `inline` is a size (`1em` vs the other size variations which are specific `rem` sizes), so it made more sense to me for it to be a size variation since you wouldn't want to combine `size` and `isInline`. I couldn't get it to work though, and I saw the icon component was setup as `isInline`, so I went that way.

Related, I'm not sure why we default to an `xl` spinner by default, I would probably expect `inline` to be the default size - it is sized relative to where it's used.